workspace/utilities/com/com-embed-video.xsl

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: com-embed-video -->
+<xsl:template name="com-embed-video">
+	<xsl:param name="thumbnail" select="thumbnail" />
+	<xsl:param name="custom-thumbnail" />
+	<xsl:param name="is-autoplay" select="true()" />
+	<xsl:param name="autoload" select="'none'" />
+	<xsl:param name="title" select="''" />
+	<xsl:param name="ext-attr" />
+	<xsl:param name="ext-attr-video" />
+	<xsl:param name="ext-attr-placeholder" />
+	<xsl:param name="ext-attr-thumbnail" />
+	<xsl:param name="ext-attr-button-ctn" />
+
+
+	<xsl:variable name="has-thumbnail" select="string-length($thumbnail) != 0" />
+	<xsl:variable name="has-custom-thumbnail" select="string-length($custom-thumbnail) != 0" />
+
+	<!-- ATTRIBUTES -->
+	<xsl:variable name="attr">
+		<add class="padding-bottom-9_16 relative z-index-1" />
+		<add class="bg-color-black" />
```

Comment:
Should this be really set by default on the ctn ?

Reply:
I don't have a strong opinion about this. I tend to prefer having a smart default, that uses generated classes anyway. It should probably be bg-color-true-black now that I look at it. And maybe split style classes from layout classes.

@alexnantel88 @fhamon What do you think?

Reply:
@dominic-blain Ok with me

Reply:
Updated